.github/workflows/runtime_ci.yml

```diff
@@ -0,0 +1,68 @@
+name: Runtime Testing
```

Comment:
can we cherry-pick https://github.com/darshan-hpc/darshan/pull/753/commits/c539ef14e35893bb529f5c41b7249deaf574e68b here?

Reply:
scratch that, i figured out how to push directly to the PR